### ISSUE TITLE
Technical issues running BIMAN

### ISSUE THREAD

**Issue by mairieli** [ssc-oscar/BIMAN_bot_detection#1]
Hi @tapjdey, 

I am facing some technical issues trying to run the BIMAN approach. I am following the tutorial provided in the README.md file.

1.  First, I created a file called "bots.txt" with the same content as the example in the tutorial. 
2. Then, I executed `./BIN.sh bots.txt`, which generated the following output:

![botsss](https://user-images.githubusercontent.com/5549736/103832886-374eef80-505e-11eb-87d4-ca9a799c274d.png)

I couldn't see any other output when running this command.  Which is the expected outcome in this case? 

I hope you could help me. Let me know if you need any other details.

Thanks in advance!

**Comment by tapjdey** [comment]
Hi @mairieli 
Yes, that is the expected output. 
Quoting from our paper:

> regular expressions were used to identify if an author is a bot by checking if the author's name or email has the substring “bot”. However, to avoid including false positives like “Abbot” or “Botha”, the regular expression searched for “bot” preceded and followed by non-alpha characters. We further excluded author IDs that had the word “bot” only in the domain name of their email addresses (e.g., hr@future-bot.ai), since we are not convinced that these are always bots. 

If you check the list of names in the example list,

```
dependabot[bot] <support@dependabot.com>
greenkeeper <felixhennigfh@gmail.com>
Greenkeeper <support@greenkeeper.io>
greenkeeper.io <support@greenkeeper.io>
greenkeeper-keeper <nodaguti+keeper@gmail.com>
greenkeeper <greenkeeper@users.noreply.github.com>
Greenkeeper <green@fxck.it>
Greenkeeper <support@greenkeeper.io>
SPOT robot <spotbot@ebi-cli-003.ebi.ac.uk>
```

Only the first one has the name "bot" in it, so it is the only one detected by BIN as a bot.
You might be confused since the name "greenkeeper" appears multiple times, but the string "bot" is not present in any of them. For "SPOT robot", the string "bot" doesn't appear by itself, so it's similar to "Abbot", and therefore isn't picked up by BIN. That's why you'll need the other two methods to complement the results by BIN.

In hindsight, I suppose I should have chosen a less confusing example :) 

In summary, yes the script works as expected!
